src/Randomizer.Shared/Models/TrackerDungeonState.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Randomizer.Shared.Models {
+
+    public class TrackerDungeonState
+    {
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+        public int TrackerStateId { get; set; }
```

Comment:
What is this value?

Reply:
Just foreign key to TrackerState object.

Reply:
Oh. Please add a navigation property (`public TrackerState TrackerState {get; set;}`) to clear that up, then. Also makes it easier to use.